developer_neuron_id.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
+
+set -euo pipefail
+
+. ./constants.sh normal
+
+echo "$(dfx canister --network "${NETWORK}" call sns_governance list_neurons "(record {of_principal = opt principal\"${DFX_PRINCIPAL}\"; limit = 1})" | grep "^ *id = blob" | sed "s/^ *id = \(.*\);$/'(\1)'/" | xargs didc encode | tail -c +21)"
```

Comment:
Nit: I would prefer to use `didc` and `jq` to select fields from inherently structured data.

Reply:
Using `idl2json`, `jq`, and `python3` to derive SNS neuron IDs now.